### PR TITLE
Use SonataTranslationBundle instead of CmfCoreBundle

### PR DESCRIPTION
**Pull request by WouterJ**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
         "sonata-project/cache-bundle": "^2.1",
         "sonata-project/doctrine-phpcr-admin-bundle": "1.2.*",
         "sonata-project/admin-bundle": "^2.3.3",
+        "sonata-project/translation-bundle": "1.0.0",
```
Comment (WouterJ) [inline composer.json]: Like all Sonata packages, `1.0.1` contained a major BC break and is not usable with SonataBlockBundle 2.2.

Comment (dbu) [inline composer.json]: do we lift things to after that BC break for 1.3?

**Comment by dbu** [comment]
great, thanks!

there is a conflict, styleci complaints and the build fails. can you have a look at that please?

**Comment by WouterJ** [comment]
Everything is green now. Had to include some polyfill as the polyfill in the Symfony package was plain wrong and causing failures (see also https://github.com/symfony/polyfill/pull/31 ).

File: composer.json (hunk 1 above)
Comment (WouterJ) [inline composer.json]: I'm sorry, I don't really understand what you try to ask me.

Comment (dbu) [inline composer.json]: you commented that sonata 1.0.1 has a major BC break. i was wondering when we plan to move to that and stop supporting the old version. can we do that before we release 1.3 or is it a big thing?

**Comment by dbu** [comment]
:+1:

**Pull request merged**